### ISSUE TITLE
schemaorg-process-schema subcommand failed

### ISSUE THREAD

**Issue by inbaroren** [stanford-oval/genie-toolkit#401]
Running the command below (from the starter/schemaorg/Makefile) failed:
`node --experimental_worker --max_old_space_size=15000 ../../tool/genie.js schemaorg-process-schema -o restaurants/schema.tt --manual --domain restaurants --class-name org.schema --white-list Restaurant,Review`

Error:
```
Triple failed
{ '@id': 'schema:healthPlanCostSharing',
  '@type': [ 'rdf:Property' ],
  'rdfs:comment':
   'Whether The costs to the patient for services under this network or formulary.',
  'rdfs:label': 'healthPlanCostSharing',
  'schema:domainIncludes':
   [ { '@id': 'schema:HealthPlanNetwork' },
     { '@id': 'schema:HealthPlanFormulary' } ],
  'schema:isPartOf': { '@id': 'http://pending.schema.org' },
  'schema:rangeIncludes': { '@id': 'schema:Boolean' },
  'schema:source':
   { '@id': 'https://github.com/schemaorg/schemaorg/issues/1062' } }
/genie-toolkit/tool/genie.js:21
process.on('unhandledRejection', (up) => { throw up; });
                                           ^

TypeError: Cannot read property '@id' of undefined
    at getIncludes (/genie-toolkit/tool/autoqa/schemaorg/process-schema.js:74:31)
    at SchemaProcessor.run (/genie-toolkit/tool/autoqa/schemaorg/process-schema.js:498:41)
```
I was able to solve this by removing the cached file, and passing the url of version 9.0: `--url https://raw.githubusercontent.com/schemaorg/schemaorg/main/data/releases/9.0/schemaorg-current-http.jsonld`

(there's a [FIXME note](https://github.com/stanford-oval/genie-toolkit/blob/master/tool/autoqa/schemaorg/process-schema.js#L807) in the code to change the default url)

**Comment by rayslxu** [comment]
> Running the command below (from the starter/schemaorg/Makefile) failed:
> `node --experimental_worker --max_old_space_size=15000 ../../tool/genie.js schemaorg-process-schema -o restaurants/schema.tt --manual --domain restaurants --class-name org.schema --white-list Restaurant,Review`
> 
> Error:
> 
> ```
> Triple failed
> { '@id': 'schema:healthPlanCostSharing',
>   '@type': [ 'rdf:Property' ],
>   'rdfs:comment':
>    'Whether The costs to the patient for services under this network or formulary.',
>   'rdfs:label': 'healthPlanCostSharing',
>   'schema:domainIncludes':
>    [ { '@id': 'schema:HealthPlanNetwork' },
>      { '@id': 'schema:HealthPlanFormulary' } ],
>   'schema:isPartOf': { '@id': 'http://pending.schema.org' },
>   'schema:rangeIncludes': { '@id': 'schema:Boolean' },
>   'schema:source':
>    { '@id': 'https://github.com/schemaorg/schemaorg/issues/1062' } }
> /genie-toolkit/tool/genie.js:21
> process.on('unhandledRejection', (up) => { throw up; });
>                                            ^
> 
> TypeError: Cannot read property '@id' of undefined
>     at getIncludes (/genie-toolkit/tool/autoqa/schemaorg/process-schema.js:74:31)
>     at SchemaProcessor.run (/genie-toolkit/tool/autoqa/schemaorg/process-schema.js:498:41)
> ```
> 
> I was able to solve this by removing the cached file, and passing the url of version 9.0: `--url https://raw.githubusercontent.com/schemaorg/schemaorg/main/data/releases/9.0/schemaorg-current-http.jsonld`
> 
> (there's a [FIXME note](https://github.com/stanford-oval/genie-toolkit/blob/master/tool/autoqa/schemaorg/process-schema.js#L807) in the code to change the default url)

Thank you for testing out our code. 
In fact we did fix this problem in our working branch (467531c). We will do a pr to fix it in master branch soon.

**Comment by gcampax** [comment]
This was fixed a long time ago.
